.github/workflows/integration-tests.yml

```diff
@@ -483,7 +483,7 @@ jobs:
 
             -   uses: ./.github/actions/teardown
 
-    CITestsZosmfRsu2012:
+    CITestsZosmfPH12143:
```

Comment:
you can remove all zosmf specific test runs, the only reason why there were these tests was different supporting versions, now there is just 1 so it can run together with the rest of the tests, e.g. Zaas tests

Reply:
I did not delete CITestsZosmfPH12143 , because it has a different configuration compares to zaasTest and we have many test classes with a tag ZosmfAuthTest which run as part of CITestsZosmfPH12143.